src/Test/L0/RunnerWebProxyL0.cs

```diff
@@ -351,7 +351,7 @@ public void WebProxyFromEnvironmentVariablesNoProxy()
                 Assert.False(proxy.IsBypassed(new Uri("https://actions.com")));
                 Assert.False(proxy.IsBypassed(new Uri("https://ggithub.com")));
                 Assert.False(proxy.IsBypassed(new Uri("https://github.comm")));
-                Assert.False(proxy.IsBypassed(new Uri("https://google.com")));
+                Assert.False(proxy.IsBypassed(new Uri("https://google.com"))); // no_proxy has '.google.com', specifying only subdomains bypass
```

Comment:
Not a change in behaviour in this PR. It's an additional test clarifying existing behaviour.